package-lock.json

```diff
@@ -6720,56 +6622,36 @@
         "globule": "^1.0.0"
       }
     },
-    "gcp-metadata": {
-      "version": "3.5.0",
-      "resolved": "https://registry.npmjs.org/gcp-metadata/-/gcp-metadata-3.5.0.tgz",
-      "integrity": "sha512-ZQf+DLZ5aKcRpLzYUyBS3yo3N0JSa82lNDO8rj3nMSlovLcz2riKFBsYgDzeXcv75oo5eqB2lx+B14UvPoCRnA==",
-      "requires": {
-        "gaxios": "^2.1.0",
-        "json-bigint": "^0.3.0"
-      }
-    },
-    "gcs-resumable-upload": {
-      "version": "2.3.3",
-      "resolved": "https://registry.npmjs.org/gcs-resumable-upload/-/gcs-resumable-upload-2.3.3.tgz",
-      "integrity": "sha512-sf896I5CC/1AxeaGfSFg3vKMjUq/r+A3bscmVzZm10CElyRanN0XwPu/MxeIO4LSP+9uF6yKzXvNsaTsMXUG6Q==",
-      "requires": {
-        "abort-controller": "^3.0.0",
-        "configstore": "^5.0.0",
-        "gaxios": "^2.0.0",
-        "google-auth-library": "^5.0.0",
-        "pumpify": "^2.0.0",
-        "stream-events": "^1.0.4"
-      }
-    },
     "genome-visualizer": {
-      "version": "git://github.com/ENCODE-DCC/valis-hpgv.git#413ff3f2c16a026a555ea0edf1cb64e3204d044e",
-      "from": "git://github.com/ENCODE-DCC/valis-hpgv.git#1.5.0",
+      "version": "git://github.com/ENCODE-DCC/valis-hpgv.git#84bd9586f98fccef4cc5f3725af62ebd87ee845f",
```

Comment:
There seem like there are a lot of package-lock changes here. Don't we just want this one package updated? This line looks great though - this is the change that we definitely need.

Reply:
Good point. npm updated these packages itself. Many packages are set up to continuous update minor and patch updates but not major; when you do npm install.